packages/analytics-core/src/index.ts

```diff
@@ -4,7 +4,7 @@ export { Revenue } from './revenue';
 export { Destination } from './plugins/destination';
 export { Config } from './config';
 export { Logger } from './logger';
-export { AMPLITUDE_PREFIX } from './constants';
+export { AMPLITUDE_PREFIX, STORAGE_PREFIX } from './constants';
```

Comment:
Thanks @falconandy. I think we should also update `amplitude-types`

Reply:
Updated